.github/workflows/linux.yml

```diff
@@ -784,9 +849,9 @@ jobs:
 
       - name: Intel CPU plugin func tests (parallel)
         run: |
-          source ${{ env.INSTALL_DIR }}/setupvars.sh
-          python3 ${{ env.PARALLEL_TEST_SCRIPT }} -e ${{ env.INSTALL_TEST_DIR }}/ov_cpu_func_tests -c ${{ env.PARALLEL_TEST_CACHE }} -w ${{ env.INSTALL_TEST_DIR }} -s suite -rf 0 -- --gtest_print_time=1 --gtest_filter=*smoke*
-        timeout-minutes: 25
+          source ${INSTALL_DIR}/setupvars.sh
+          python3 ${PARALLEL_TEST_SCRIPT} -e ${INSTALL_TEST_DIR}/ov_cpu_func_tests -c ${PARALLEL_TEST_CACHE} -w ${INSTALL_TEST_DIR} -s suite -rf 0 -- --gtest_print_time=1 --gtest_filter=*smoke*
+        timeout-minutes: 40
```

Comment:
why the execution time has been increased

Reply:
I've found out the root cause and this is expected behavior, cpu_func_tests silently skip some tests depends on hardware